python/dolfinx/io/gmshio.py

```diff
@@ -292,158 +300,96 @@ def model_to_mesh(
         x = extract_geometry(model)
         topologies, physical_groups = extract_topology_and_markers(model)
 
-        # Extract Gmsh cell id, dimension of cell and number of nodes to
-        # cell for each
+        # Extract Gmsh cell id, dimension of cell and number of nodes in cell
         num_cell_types = len(topologies.keys())
-        cell_information = dict()
-        cell_dimensions = np.zeros(num_cell_types, dtype=np.int32)
+        element_ids = np.zeros(num_cell_types, dtype=np.int32)
```

Comment:
Should this be renamed to cell_ids, etc. below?

Reply:
I am not sure, as it is more than just a cell (type), it is a coordinate element. I'd rather rename `num_cell_types` to `num_coordinate_elements`.